src/Microsoft.AspNet.Mvc.Core/ModelBinding/FormValueProviderFactory.cs

```diff
@@ -20,7 +20,7 @@ public class FormValueProviderFactory : IValueProviderFactory
 
                 return new ReadableStringCollectionValueProvider(
                     BindingSource.Form,
-                    await request.ReadFormAsync(),
+                    await request.ReadFormAsync().ConfigureAwait(false),
                     culture);
```

Comment:
Culture call suggests is retrieved from request; but call gets it from current thread?